dashboard/app/assets/stylesheets/application.scss

```diff
@@ -345,20 +345,58 @@ img.video_thumbnail {
 }
 
 #header-banner {
-  top: 0;
-  position: absolute;
-  width: 100%;
-  height: 320px;
-  z-index: -1000;
-  background-position: 90% 10%;
   background-size: cover;
+  background-position: 90% 30%;
+  display: flex;
+  justify-content: center;
+
+  .bannerContent {
+    max-width: 60%;
+  }
+
+  /*
+    The banner background image leaves an open space for text on the left. Therefore, in Right-to-Left languages, the
+    banner needs to be flipped horizontally so the background image empty space is on the right.
+   */
+  html[dir=rtl] & {
+    transform: scaleX(-1);
+    /*
+      Since the whole banner was flipped horizontally, we need to un-flip all the content in the header. So the text and
+      layout is not backwards
+     */
+    .bannerContentContainer {
```

Comment:
Is it possible to flip only the image and not its content, so that the unflip is not needed?  I don't know much about this, but found an answer that alludes to such a possibility at https://stackoverflow.com/a/5769230.

Reply:
I can't figure out a simple way to do this given that the background image is defined in a jsx file and not the css files. I personally feel that the solution proposed in the stackoverflow is more complicated and verbose than flipping twice. I'm open to discussing this further though.